user/test-super/com/google/gwt/dev/jjs/super/com/google/gwt/dev/jjs/test/Java9Test.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2023 GwtProject contributors
```

Comment:
Odd that it doesn't mind this header.

Google has no interest in this code any more, as we no longer need a CLA. I'm happy to change this PR for the sake of landing it now, but any thoughts on relaxing this requirement, allow any apache2 code, encourage a more general copyright notice?

Reply:
+1 on relaxing the requirement. How about changing everything to "The GWT Project authors" and adding an AUTHORS file?
(see https://opensource.google/documentation/reference/releasing/authors; that's also what I did in the few "modularized gwt-user" modules, populating the AUTHORS file from the Git history)